src/Sync.ts

```diff
@@ -0,0 +1,200 @@
+        synchronizedFlicking.flicking.off(EVENTS.MOVE_END, this._onMoveEnd);
+      }
+      if (this._type === "index" && synchronizedFlicking.isSlidable) {
+        synchronizedFlicking.flicking.off(EVENTS.WILL_CHANGE, this._onChangeIndex);
+      }
+      if (this._type === "index" && synchronizedFlicking.isClickable) {
+        synchronizedFlicking.flicking.off(EVENTS.SELECT, this._onChangeIndex);
+      }
+    });
+  };
+
+  private _onChangeIndex = (e: ComponentEvent) => {
+    const flicking = e.currentTarget;
+    const synchronizedFlickings = this._synchronizedFlickings;
+
+    if (!flicking.initialized || synchronizedFlickings.reduce((isAnimationComplete, synchronizedFlicking) => isAnimationComplete || (synchronizedFlicking.flicking !== flicking && synchronizedFlicking.flicking.animating), false)) {
+      return;
+    }
+    
+    this._synchronizeByIndex(flicking, e["index"]);
+  };
+
+  private _onMove = (e: ComponentEvent): void => {
+    const camera = e.currentTarget.camera;
+    const progress = (camera.position - camera.range.min) / camera.rangeDiff;
+
+    this._synchronizedFlickings.forEach(synchronizedFlicking => {
+      if (synchronizedFlicking.flicking !== e.currentTarget) {
+        synchronizedFlicking.flicking.camera.lookAt(synchronizedFlicking.flicking.camera.range.min + synchronizedFlicking.flicking.camera.rangeDiff * progress);
+      }
+    });
+  };
+
+  private _onMoveStart = (e: ComponentEvent): void => {
+    this._synchronizedFlickings.forEach(synchronizedFlicking => {
+      if (synchronizedFlicking.flicking !== e.currentTarget) {
+        synchronizedFlicking.flicking.disableInput();
+      }
+    });
+  };
+
+  private _onMoveEnd = (e: ComponentEvent): void => {
+    this._synchronizedFlickings.forEach(synchronizedFlicking => {
+      if (synchronizedFlicking.flicking !== e.currentTarget) {
+        synchronizedFlicking.flicking.enableInput();
+        synchronizedFlicking.flicking.control.updateInput();
+      }
+    });
+  };
+  
+  private _synchronizeByIndex = (flicking: Flicking, index: number) => {
+    const synchronizedFlickings = this._synchronizedFlickings;
+    const activePanel =  flicking.panels.find(panel => panel.index === index);
+    const lastPanel = flicking.panels[flicking.panels.length - 1];
+
+    if (!activePanel) {
+      return;
+    }
+
+    this._removeEvents(synchronizedFlickings);
+    synchronizedFlickings.forEach((synchronizedFlicking) => {
+      const targetLastPanel = synchronizedFlicking.flicking.panels[synchronizedFlicking.flicking.panels.length - 1];
+      const targetPos = activePanel.position / (lastPanel.position + (lastPanel.size / 2)) * (targetLastPanel.position + (targetLastPanel.size / 2));
+
+      synchronizedFlicking.flicking.control.moveToPosition(targetPos, 500);
+      if (synchronizedFlicking.activeClass) {
+        this._updateClass(synchronizedFlicking, targetPos);
+      }
+    });
+    this._addEvents(synchronizedFlickings);
```

Comment:
![3](https://user-images.githubusercontent.com/26213435/125376957-124d8780-e3c7-11eb-95df-3a3c485de858.gif)

Flicking is unable to respond to the continuous index changes. It would be better if there is an improvement on this part.

---

현재는 위와 같이 연속된 인덱스 변경에 대응하지 못하고 있는데요, 이 부분에 대한 개선이 있으면 좋을 것 같네요.

Reply:
현재는 `Animation is interrupted by user input` 오류 메시지를 방지하기 위하여 애니메이션이 재생 중일 때는 moveToPosition을 실행하지 않게끔 하였으나, 이로 인해 남겨주신 문제가 발생하는 것 같습니다. 애니메이션 재생과 관련된 조건이 제거된 근의 커밋 이후에도 해당 오류가 재현되는지 확인이 가능할까요?

Currently, to prevent the `Animation is interrupted by user input` error message, moveToPosition is not executed while animation is playing, but this seems to cause the problem you left. Is it possible to check if the error is reproduced even after the root commit with the condition related to animation playback removed?

Reply:
It works as intended! But it shows error messages, which can be easily silenced.

```diff
- synchronizedFlickingOption.flicking.control.moveToPosition(targetPos, 500);
+ synchronizedFlickingOption.flicking.control.moveToPosition(targetPos, 500).catch(() => void 0);
```

Reply:
Thank you very much. Error messages has silenced in recent commit.